pkg/sql/codegen/xgboost/codegen.go

```diff
@@ -203,6 +204,41 @@ func resolveFeatureMeta(fds []ir.FieldDesc) ([]byte, []string, error) {
 	return f, featureNames, e
 }
 
+func generateFeatureColumnCode(fc ir.FeatureColumn) (string, error) {
+	return codegen.GenerateFeatureColumnCode(fc, "xgboost_extended")
+}
+
+func deriveFeatureColumnCode(fcs []ir.FeatureColumn, labelFc ir.FeatureColumn) (featureColumnsCode string, fieldDescs []ir.FieldDesc, label ir.FieldDesc, err error) {
```

Comment:
How about add more description on this method？

Reply:
Added.